examples/astro-forms-demo/src/components/URLInput.tsx

```diff
@@ -0,0 +1,55 @@
+import { type UrlInputProps } from '@wix/headless-forms/react';
+import {
+  quickStartViewerPlugins,
+  RicosViewer,
+  type RichContent,
+} from '@wix/ricos';
+import '@wix/ricos/css/all-plugins-viewer.css';
+
+const URLInput = ({
```

Comment:
nitpick: lets use same capitalization syle as in props: UrlInputProps
```suggestion
const UrlInput = ({
```